homeassistant/components/yolink/entities.py

```diff
@@ -0,0 +1,108 @@
+"""Entity for YoLink devices."""
+from homeassistant.components.binary_sensor import BinarySensorEntity
```

Comment:
Can you remove unused types from this PR? And it would be better if you just move the relevant sensor entities to your sensor.py.

Reply:
All types are used